phost/src/chain_client.rs

```diff
@@ -136,6 +136,13 @@ pub fn check_round_end_event(
     Ok(false)
 }
 
+pub async fn get_parachain_heads(client: &XtClient, paraclient: &XtClient, hash: Option<Hash>) -> Option<Vec<u8>> {
+	let para_key = storage_value_key_vec("ParachainInfo", "ParachainId");
+	let para_id = get_storage(&paraclient, None, StorageKey(para_key)).await.unwrap().unwrap();
+	let key = storage_map_key_vec("Paras", "Heads", &hex::encode(para_id));
+	get_storage(&client, hash, StorageKey(key)).await.unwrap()
```

Comment:
`get_storage` returns the raw storage in `Vec<u8>`. The data in the vec is `Para.Heads`, which is also a `Vec<u8>`. So we should decode it by:

```
match get_storage(&client, hash, StorageKey(key)).await {
    Ok(Some(raw)) => Some(Vec::<u8>::decode(&mut &raw))
    _ => None
}
```

Reply:
i'm not sure